cvmfs/catalog_mgr.h

```diff
@@ -373,8 +401,12 @@ class AbstractCatalogManager : public SingleCopy {
     assert(retval == 0);
   }
   inline void WriteLock() const {
+    uint64_t timestamp = platform_monotonic_time_ns();
```

Comment:
i guess we want to keep those timers?

Reply:
With `n_write_lock` and `ns_write_lock` we get the average time spent in `WriteLock()`. If we need the distribution, we can use the [Log2Histogram|https://github.com/cvmfs/cvmfs/blob/devel/cvmfs/util/algorithm.h#L141]